common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -63,11 +63,6 @@ public interface @K@ObjectMap<V> {
     V put(@k@ key, V value);
 
     /**
-     * Puts all of the entries from the given map into this map.
-     */
-    void putAll(@K@ObjectMap<V> sourceMap);
```

Comment:
@nmittler - Since this involves iterator over the map....and we have seen there may be benefits to using the "primitive" iterator... should we keep this in the interface?